bin/rose-make-docs

```diff
@@ -179,10 +179,10 @@ venv-install () {
     venv-activate
     # TODO remove Sphinx version dependency
```

Comment:
This TODO has just been addressed, so can be removed I think (:ballot_box_with_check:)?